spec/models/general_info_spec.rb

```diff
@@ -5,6 +5,13 @@
     let(:incident) { create(:incident) }
     let(:gi) { incident.general_info }
 
+    def create_duplicate
```

Comment:
minor: I would put a `!` in the method name since it's modifies state. Although I guess we haven't really had strong conventions for `!` methods in our specs.